packages/data/addon/adapters/facts/elide.ts

```diff
@@ -63,6 +63,12 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
     const filterStrings = filters.map(filter => {
       const { field, parameters, operator, values } = filter;
       const fieldStr = getElideField(field, parameters);
+
+      // TODO: Remove this when Elide supports the "between" filter operator
+      if (operator === 'bet') {
+        return `${fieldStr}=ge=(${values[0]});${fieldStr}=lt=(${values[1]})`;
```

Comment:
between is inclusive on both sides:
```suggestion
        return `${fieldStr}=ge=(${values[0]});${fieldStr}=le=(${values[1]})`;
```